internal/blueprint/customizations.go

```diff
@@ -74,6 +74,7 @@ type UserCustomization struct {
 	Groups      []string `json:"groups,omitempty" toml:"groups,omitempty"`
 	UID         *int     `json:"uid,omitempty" toml:"uid,omitempty"`
 	GID         *int     `json:"gid,omitempty" toml:"gid,omitempty"`
+	ExpireDate  *int     `json:"expiredate,omitempty" toml:"expiredate,omitempty"`
```

Comment:
Again, separate PR? I don't know if @andremarianiello wants to take this but I understand it's blocking?

Reply:
yes it's blocking, I'm happy to open a new PR with v51